File: dart/lib/model/v3_outlet_parameters.dart
part of swagger.api;

class V3OutletParameters {
  /* Maximum number of results returned (default = 30) */
  int maxResults = null;

  V3OutletParameters();

  @override
  String toString() {
    return 'V3OutletParameters[maxResults=$maxResults, ]';
  }

  V3OutletParameters.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    maxResults = json['max_results'];
  }

  Map<String, dynamic> toJson() {
    return {
      'max_results': maxResults
     };
  }

  static List<V3OutletParameters> listFromJson(List<dynamic> json) {
    return json == null ? new List<V3OutletParameters>() : json.map((value) => new V3OutletParameters.fromJson(value)).toList();
  }

  static Map<String, V3OutletParameters> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, V3OutletParameters>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new V3OutletParameters.fromJson(value));
    }
    return map;
  }
}

File: csharp-dotnet2/src/main/CsharpDotNet2/IO/Swagger/Model/V3OutletParameters.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class V3OutletParameters {
    /// <summary>
    /// Maximum number of results returned (default = 30)
    /// </summary>
    /// <value>Maximum number of results returned (default = 30)</value>
    [DataMember(Name="max_results", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "max_results")]
    public int? MaxResults { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class V3OutletParameters {\n");
      sb.Append("  MaxResults: ").Append(MaxResults).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
